app/usecase/feedback_interactor.go

```diff
@@ -30,3 +30,20 @@ func (interactor *FeedbackInteractor) StoreFeedback(feedback domain.Feedback) (s
 	success, err = interactor.FeedbackRepository.InsertFeedback(userId, feelingId, categoryId, recipientId, feedback.NewsId, feedback.FeedbackNote)
 	return
 }
+func (interactor *FeedbackInteractor) ChangeToSeen(ids []string) (numberOfChanged int, err error) {
+	query := createIdsQuery(ids)
+	numberOfChanged, err = interactor.FeedbackRepository.UpdateSeen(query)
+	return
+}
+
+func createIdsQuery(ids []string) (query string) {
+	query = "("
+	for i, id := range ids {
+		if i != len(ids)-1 {
+			query += id + ", "
+		} else {
+			query += id + ")"
+		}
+	}
+	return
+}
```

Comment:
For this point, you get an array of strings, and my understanding that you need to change from
["string1", "string2", "string3"] to
"string1, string2, string3" Correct??
Do you want to leave this iterative approach, 
Or perhaps do a `strings.Join(ids, ", ")` -> to collect everything into 1 string and then add the "(" and ")" ??

Reply:
I'll do it in another PR.